src/room/RoomPage.tsx

```diff
@@ -82,82 +85,124 @@ export const RoomPage: FC = () => {
     if (optInAnalytics === null && setOptInAnalytics) setOptInAnalytics(true);
   }, [optInAnalytics, setOptInAnalytics]);
 
-  const groupCallView = useCallback(
-    (groupCallState: GroupCallStatus): JSX.Element => {
-      switch (groupCallState.kind) {
-        case "loaded":
+  const wasInWaitForInviteState = useRef<boolean>(false);
+
+  useEffect(() => {
+    if (groupCallState.kind === "loaded" && wasInWaitForInviteState) {
+      logger.log("Play join sound 'Not yet implemented'");
+    }
+  }, [groupCallState.kind]);
+
+  const groupCallView = useCallback((): JSX.Element => {
```

Comment:
Since `groupCallView` isn't being used as a reactive dependency for anything, a bare function would work just as well as `useCallback`